intelmq/bots/collectors/shadowserver/collector_reports_api.py

```diff
@@ -48,11 +53,12 @@ def init(self):
             raise ValueError('No api_key provided.')
         if self.secret is None:
             raise ValueError('No secret provided.')
-        if self.country is None:
-            raise ValueError('No country provided.')
 
-        if isinstance(self.types, str):
-            self.types = self.types.split(',')
+        if isinstance(self.reports, str):
+            self.report_list = self.reports.split(',')
```

Comment:
What if self.reports is not a string, but e.g. a list? self.report_list stays None?

Reply:
Fixed:
```
        if isinstance(self.reports, str):
            self._report_list = self.reports.split(',')
        elif isinstance(self.reports, list):
            self._report_list = self.
```